docs/UPGRADING.rst

```diff
@@ -55,6 +55,9 @@ Release Notes
   3. Update the HIL software as usual. Before restarting the serivces, remove
      both of the above obm extensions from ``hil.cfg``.
 
+* OpenVSwitch is now supported in HIL.
```

Comment:
I might be more explicit here that the new driver is very much not intended for production use. (I might not even mention it, since it's intended for development).